docs/self-hosted/deployment/cloudron.md

```diff
@@ -1,8 +1,6 @@
 ---
 sidebar_label: "Cloudron"
 title: "Cloudron Chatwoot deployment guide"
-help_discord: "https://discord.gg/jypY57m"
-help_github_discussions: "https://github.com/chatwoot/chatwoot/discussions/categories/self-hosted"
```

Comment:
You can update this link instead of updating it at [here](https://github.com/chatwoot/docs/pull/256/files#diff-83c5153400eaf343b9c13e101cc3562d385bac9943b721f71bf6889f013f6a3fR63)

Reply:
The point in this PR is so this shows at the bottom of **every** page.